doc/whats_new.rst

```diff
@@ -131,6 +131,9 @@ Enhancements
    - Add option to show ``indicator features`` in the output of Imputer.
      By `Mani Teja`_.
 
+   - Reduce the memory usage of :func:`utils.mean_variance_axis` and  :func:`utils.incr_mean_variance_axis` 
```

Comment:
So I was hoping we could say that this reduced the memory usage of some estimators, rather than talking about utils here. Is that wrong? Or do we need to do more to reduce estimator memory usage?